### DISCUSSION TITLE
[Issue] your Takeout doesn't have any "year folders"

### DISCUSSION THREAD

**Discussion by sohammehta1995** [TheLastGimbus/GooglePhotosTakeoutHelper#206] [I NEED HELP] [answered]
I tried following the entire readme steps. 
1. I downloaded the photos from takeout (Only the albums I wanted). 
2. I then added each "Takeout" folder into a common "Takeout" folder
3. I ran the script as mentioned 
<img width="453" alt="Screen Shot 2023-05-07 at 3 38 51 PM" src="https://user-images.githubusercontent.com/9027490/236705933-5b414aa0-a288-4268-9369-584544e7c7d5.png">

It keeps giving me this error
<img width="906" alt="Screen Shot 2023-05-07 at 3 39 47 PM" src="https://user-images.githubusercontent.com/9027490/236705953-88ed40f8-2b47-4601-85ca-13d24fb54c27.png">

Note: I manually renamed each extracted zip to be the same name as downloaded zip file name (instead of the generic Takeout name). But the issue is still persisting

**Comment by TheLastGimbus** [comment, marked as answer]
Look at the visual guides i have drawn in the readme - you need to kinda have a reverse folder structure than you currently have - then i think it will work

**Comment by asaran2** [comment]
The issue is persisting for me as well @TheLastGimbus , it is the same error
![ss](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/32207049/2867a29c-1f24-430c-b29d-b2c21d4196b1)
I believe I have the right folder strucure. Like OP I also selected a few albums that I wanted to export
@sohammehta1995 were you ever able to get it to work for you?

**Reply by TheLastGimbus** [reply]
Show the actual strucuture instead of single screenshot
